src/sentry/killswitches.py

```diff
@@ -20,10 +22,45 @@
 Context = Dict[str, Any]
 
 
+def _update_project_configs(
+    old_option_value: Sequence[Mapping[str, Any]], new_option_value: Sequence[Mapping[str, Any]]
+) -> None:
+    """Callback for the relay.drop-transaction-metrics kill switch.
+    On every change, force a recomputation of the corresponding project configs
+    """
+    from sentry.tasks.relay import schedule_invalidate_project_config
+
+    old_project_ids = {ctx["project_id"] for ctx in old_option_value}
+    new_project_ids = {ctx["project_id"] for ctx in new_option_value}
+
+    # We want to recompute the project config for any project that was added
+    # or removed
+    changed_project_ids = old_project_ids ^ new_project_ids
+
+    with click.progressbar(changed_project_ids) as ids:
+        for project_id in ids:
```

Comment:
please skip over `None` here, which can happen if the user wants to kill all projects